beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -2200,6 +2200,9 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
 
         #[cfg(not(feature = "withdrawals-processing"))]
         {
+            // I assume this is just to avoid the unused variables warning
+            // when withdrawals-processing isn't enabled?
```

Comment:
Yeah, maybe we could globally allow `drop_non_drop` (in the Makefile) until the `feature`s are removed?